runtime/j9vm/javanextvmi.cpp

```diff
@@ -333,10 +335,14 @@ JVM_VirtualThreadMountEnd(JNIEnv *env, jobject thread, jboolean firstMount)
 			 * This prevents the root reference from becoming stale if the GC moves the object.
 			 */
 			j9object_t rootVirtualThread = mmFuncs->J9AllocateObject(currentThread, virtualThreadClass, J9_GC_ALLOCATE_OBJECT_NON_INSTRUMENTABLE);
+
+			J9VMJAVALANGVIRTUALTHREAD_SET_STATE(currentThread, rootVirtualThread, J9VM_VIRTUALTHREAD_ROOT_NODE_STATE);
```

Comment:
Why is the state of the fake/root virtual thread set? It should be documented in the commit message.

Reply:
its an eye catcher, so we can tell the difference between the root and a corrupt VirtualThread